hummingbot/market/coinbase_pro/coinbase_pro_api_order_book_data_source.py

```diff
@@ -114,9 +115,18 @@ def __init__(self, symbols: Optional[List[str]] = None):
 
     @property
     def order_book_class(self) -> CoinbaseProOrderBook:
+        """
+        *required
+        Get relevant order book class to access class specific methods
+        :returns: OrderBook class
+        """
         return CoinbaseProOrderBook
 
     async def get_trading_pairs(self) -> List[str]:
+        """
+        Get a list of active trading pairs
```

Comment:
maybe mention that if symbols is defined, it just returns that

Reply:
fixed.